### ISSUE TITLE
BatchNormalization Layer not added

### ISSUE THREAD

**Issue by TohnoYukine** [yashk2810/MNIST-Keras#4]
When creating a model, `model.add(BatchNormalization())` is to add the layer, while `BatchNormalization()` is to call the constructor and does nothing.

**Comment by yashk2810** [comment]
Yeah, that's right. Sorry about that. Can you create a pull request?
